Kumi.Game/Gameplay/Clocks/GameplayClockContainer.cs

```diff
@@ -0,0 +1,99 @@
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Timing;
+
+namespace Kumi.Game.Gameplay.Clocks;
+
+public partial class GameplayClockContainer : Container, IAdjustableClock, IGameplayClock
```

Comment:
This class is pretty painfully similar to [osu!lazer's own GameplayClockContainer](https://github.com/ppy/osu/blob/ed02335e3eb802c1865a653eac0af1b1050f3121/osu.Game/Screens/Play/GameplayClockContainer.cs).

Reply:
To be fair, most of the functions here are implementations of the interface, so I think we're fine on that part.

Reply:
That would only stand if the function bodies weren't nearly the same as well.

Reply:
Not sure how else you want me to change them, it's the simplest implementation imaginable. One change I could add is just to remove the bindable property of `IsPaused`, but overall everything kinda has to be kept the same.

Reply:
Likely better to just add the funny copyright comment block.

Reply:
Probably? I don't see a reason why I should add that if it's something this simple of an implementation. Even then, osu!lazer has a more [complex](https://github.com/ppy/osu/blob/master/osu.Game/Screens/Play/MasterGameplayClockContainer.cs) [implementation](https://github.com/ppy/osu/blob/master/osu.Game/Beatmaps/FramedBeatmapClock.cs) which we don't need to use.